ci-operator/step-registry/ipi/install/rbac/ipi-install-rbac-commands.sh

```diff
@@ -7,3 +7,8 @@ set -o pipefail
 # We want the cluster to be able to access these images
 oc adm policy add-role-to-group system:image-puller system:authenticated --namespace "${NAMESPACE}"
 oc adm policy add-role-to-group system:image-puller system:unauthenticated --namespace "${NAMESPACE}"
+
+function update_config_map() {
```

Comment:
Each step is run in its own pod. Defining a function in a previous step does not make the function available in future steps. I was suggesting that you create a new step that is run after the install to write to the configmap. Then add the new step to the current chains.
https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/ipi-install-chain.yaml
https://github.com/openshift/release/blob/master/ci-operator/step-registry/upi/gcp/pre/upi-gcp-pre-chain.yaml

Reply:
Hmm makes sense. How would I pass the start and end times in that case?

Reply:
All of the steps have access to the `$SHARED_DIR` directory. You would write a file to that directory in one step and read the file in another step.